packages/fxa-auth-server/lib/routes/subscriptions/paypal.ts

```diff
@@ -49,8 +64,23 @@ export const paypalRoutes = (
     stripeHelper
   );
 
-  // FIXME: All of these need to be wrapped in Stripe error handling
-  // FIXME: Many of these stripe calls need retries with careful thought about
-  //        overall request deadline. Stripe retries must include a idempotency_key.
-  return [];
+  return [
+    {
+      method: 'POST',
+      path: '/oauth/subscriptions/paypal-checkout',
+      options: {
+        auth: {
+          payload: false,
+          strategy: 'oauthToken',
```

Comment:
Is this is the correct strategy? I don't know which to pick and I'm not sure how to write a test for it.

Reply:
No test is needed for this bit.